client/qiskit_serverless/core/job.py

```diff
@@ -610,6 +614,32 @@ def get_program(
                 job_client=self,
             )
 
+    def get_jobs(self, title: str, provider: Optional[str] = None):
+        """Returns job ids executed the program based on parameters."""
+        provider, title = format_provider_name_and_title(
+            request_provider=provider, title=title
+        )
+
+        tracer = trace.get_tracer("client.tracer")
+        with tracer.start_as_current_span("program.get_by_title"):
+            response_data = safe_json_request(
+                request=lambda: requests.get(
+                    f"{self.host}/api/{self.version}/programs/get_by_title/{title}",
+                    headers={"Authorization": f"Bearer {self._token}"},
+                    params={"provider": provider},
+                    timeout=REQUESTS_TIMEOUT,
+                )
+            )
+            program_id = response_data.get("id", None)
```

Comment:
I would check if it's `None` here, Aki. And return and return an error if the function doesn't exist.